account_credit_control/data/data.xml

```diff
@@ -3,23 +3,24 @@
     <!--Email template -->
     <record id="email_template_credit_control_base" model="mail.template">
         <field name="name">Credit Control Email</field>
-        <field name="email_from">${user.company_id.email or ''}</field>
+        <field name="email_from">{{ user.company_id.email or '' }}</field>
         <field
             name="subject"
-        >Credit Control: (${object.policy_level_id.name or 'n/a'})</field>
-        <field name="partner_to">${object.get_emailing_contact().id or ''}</field>
+        >Credit Control: ({{ object.policy_level_id.name or 'n/a' }})</field>
+        <field name="partner_to">{{ (object.get_emailing_contact().id or '') }}</field>
         <field name="model_id" ref="model_credit_control_communication" />
         <field name="auto_delete" eval="False" />
-        <field name="lang">${object.contact_address_id.lang or 'en_US'}</field>
+        <field name="lang">{{ object.contact_address_id.lang or 'en_US' }}</field>
         <field name="report_template" ref="credit_control_summary" />
-        <field
-            name="body_html"
-        ><![CDATA[
-  Dear ${object.contact_address_id.name or ''}
-  <br/>
-  <br/>
-  ${object.policy_level_id.custom_mail_text | safe}
-  ]]></field>
+        <field name="body_html" type="html">
+            <div style="margin: 0px; padding: 0px;">
+                <p style="margin: 0px; padding: 0px; font-size: 13px;">
```

Comment:
Why this hardcoded styles?

Reply:
in Odoo 15.0 version this same style is used for all mail template so i prefer this.

Reply:
Hello @chienandalu , i have made changes and remove hard coded style.
can you please review and approve now